tests/generators/test_adaptative_corrector.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+import torch
+
+from diffusion_for_multi_scale_molecular_dynamics.generators.adaptive_corrector import \
+    AdaptiveCorrectorGenerator
+from diffusion_for_multi_scale_molecular_dynamics.noise_schedulers.noise_parameters import \
+    NoiseParameters
+from diffusion_for_multi_scale_molecular_dynamics.utils.basis_transformations import \
+    map_relative_coordinates_to_unit_cell
+from src.diffusion_for_multi_scale_molecular_dynamics.noise_schedulers.noise_scheduler import \
+    NoiseScheduler
+from tests.generators.test_langevin_generator import TestLangevinGenerator
+
+
+class TestAdaptativeCorrectorGenerator(TestLangevinGenerator):
```

Comment:
`Adaptative` -> `Adaptive`

Also, the word "adaptative" still apears in the main code at the following locations:

> ./src/diffusion_for_multi_scale_molecular_dynamics/noise_schedulers/noise_parameters.py:    # Step size scaling for the Adaptative Corrector Generator. Default value comes from github implementation

> ./src/diffusion_for_multi_scale_molecular_dynamics/generators/adaptive_corrector.py:    """Langevin Dynamics Generator using only a corrector step with adaptative step size for relative coordinates.

> ./src/diffusion_for_multi_scale_molecular_dynamics/generators/sde_position_generator.py:    adaptative: bool = False

> ./src/diffusion_for_multi_scale_molecular_dynamics/generators/sde_position_generator.py:                adaptive=self.sampling_parameters.adaptative,